lambdas/data-migration2/src/index.ts

```diff
@@ -2,6 +2,7 @@ import { getKnexClient } from '@cumulus/db';
 
 import { migrateExecutions } from './executions';
 import { migrateGranulesAndFiles } from './granulesAndFiles';
+import { migratePdrs } from './pdrs';
```

Comment:
We should have a basic unit test of the Lambda handler to verify that it handles migrating an execution, granule, and PDR, since that would have caught this issue in the first place

Reply:
[`41b9306` (#2140)](https://github.com/nasa/cumulus/pull/2140/commits/41b930653139bd7591eb82ed4c8b90b47e3af68e)